bootstrap/handlers/clients.go

```diff
@@ -67,6 +68,9 @@ func (cb *ClientsBootstrap) BootstrapHandler(
 
 			sp := container.SecretProviderExtFrom(dic.Get)
 			jwtSecretProvider := secret.NewJWTSecretProvider(sp)
+			if serviceInfo.SecurityOptions[bscfg.SecurityModeKey] == zerotrust.ConfigKey {
```

Comment:
Naming of value constant should be `zerotrust.ZeroTrustMode`. i.e it isn't a key...
```suggestion
			if serviceInfo.SecurityOptions[bscfg.SecurityModeKey] == zerotrust.ZeroTrustMode{
```

Reply:
I like the idea, how about SecurityMode instead since it 'SecurityModeKey'?

Reply:
actually - neermind. ZeroTrustMode makes more sense ;)